src/utils/fetchClient.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using 'eslint-disable' if possible. It's better to follow the ESLint conventions and adjust the code accordingly. This specific rule '@typescript-eslint/no-explicit-any' is important because it encourages the use of specific types instead of 'any', which can lead to type errors.